include/roboteam_ai/stp/PlayChecker.hpp

```diff
@@ -51,6 +52,11 @@ class PlayChecker {
      * Current world, do not use before update()
      */
     world::World* world{};
```

Comment:
This is not your fault, but I'm starting to see a pattern here. Now I understand why you all are so eager to do this :P